src/dxgi/dxgi_swapchain_dispatcher.h

```diff
@@ -296,6 +297,7 @@ namespace dxvk {
 
   private:
 
+    void* m_device_ptr;
```

Comment:
IUnknown* m_device;

Probably worth adding a comment as to why its not necessary to ref it either.

Reply:
Eh, that's exactly why I made it void * and suffixed _ptr, to suggest we have nothing to do with this data and specifically do not want to refcount it. Can be changed this way ofc.

Reply:
I will defer to @doitsujin's judgement then.

Reply:
Agree with Josh, we'd want that comment anyway to make it clear why we need an unused data member in the first place.